molecule/commands/base.py

```diff
@@ -32,58 +32,28 @@ class InvalidHost(Exception):
 class BaseCommand:
     __metaclass__ = abc.ABCMeta
 
-    def __init__(self, command_args, global_args, molecule=False):
+    def __init__(self, command_args, args, molecule=False):
```

Comment:
Can we remove molecule from the arg list since we're not using it when it's passed in now?